app/src/amazonWebview/java/org/mozilla/focus/webview/FirefoxWebView.kt

```diff
@@ -35,6 +36,11 @@ internal class FirefoxWebView(
     private val chromeClient: FirefoxWebChromeClient
 ) : NestedWebView(context, attrs), IWebView {
 
+    init {
+        val initialZoom = context.resources.getInteger(R.integer.webview_initial_zoom)
+        setInitialScale(initialZoom)
```

Comment:
I notice there are a few methods you didn't try out in your analysis: `WebSettings.setMinimumFontSize` for example. Do you mind taking a look through for these (in `WebSettings` and the WebViewClients) to see how that affects the content? fwiw, I appreciate the screenshots to understand each of the methods but it's a lot of work to get screenshots for all these sites so I'd recommend only doing that for your top options and providing a text description for the effects on the other ones.

Note: while the story suggests 300% as the new font size, it's possible we might find a better UX at a different zoom level. If any methods seem like they suck for 300% but might work with e.g. 200% and perform better than our options at 300%, please try them out!

Reply:
Setting the default font size had no effect on YouTube and Google Search, which are critical sites, so I don't think the other font size methods (`setMinimumFontSize`, `setMinimumLogicalFontSize`, and `setDefaultFixedFontSize`) are likely to be useful to us.  I will verify that suspicion when I am able, but I'm having trouble connecting to the device at the moment.

I'm not super familiar with the API, but I don't see any methods on the clients that seem promising.  I'm looking at `FocusWebViewClient` and `FirefoxWebChromeClient` (and of course their superclasses), are these what you meant?

Reply:
> Setting the default font size had no effect on YouTube and Google Search,

To be explicit, I'd expect default font size (if the site doesn't define a font size, use this) and minimum font size (all fonts on the page are at minimum this size, but can be larger) to be different.

> but I don't see any methods on the clients that seem promising. I'm looking at FocusWebViewClient and FirefoxWebChromeClient (and of course their superclasses), are these what you meant?

Yes. The WebView API isn't the cleanest separation (e.g. settings methods on WebView instead of WebSettings) so I wanted to check all of the major WebView APIs to see if there were relevant methods. I didn't expect much from the clients either, though. :)